### ISSUE TITLE
no picture support in CloudNotes

### ISSUE THREAD

**Issue by djm193** [owncloud/notes-iOS-App#41]
Sync works over all iOS devices. Would love to skip iOS Notes App and switch to CloudNotes. However it is not possible to add photos or pictures to a note. 

Will you implement this feature in the future?

thx
Daniel

**Comment by phedlund** [comment]
CloudNotes implements the API exposed by ownCloud Notes(https://github.com/owncloud/notes/wiki/API-0.2). The server would have to add support for images first. You can follow a current discussion here https://github.com/owncloud/notes/issues/224.

**Comment by boistordu** [comment]
no progress about it I guess?

**Comment by phedlund** [comment]
Correct, Nextcloud Notes does not support pictures. You can of course insert web links in your markdown but there is no way to sync locally stored pictures and link to them from a note.
